stylesheet/style.css

```diff
@@ -238,9 +234,111 @@ header {
   height: 45px;
   color: #edecff;
   font-size: 12px;
-  font-family: 'Roboto', sans-serif;
+  font-family: "Roboto", sans-serif;
   margin-left: 30px;
   margin-top: 30px;
   padding: 10px;
   border-color: #7d73fc;
 }
+
+.contact-form-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.contact-texts {
+  text-align: center;
+  margin-left: 10px;
+  margin-right: 10px;
+  margin-top: -4px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  color: #f5f5ff;
+}
+
+.contacts-title {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 32px;
+  color: #9390ff;
+}
+
+form {
+  list-style-type: none;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  margin-left: 10px;
+  margin-right: 10px;
+  margin-top: 30px;
+}
+
+form > input {
+  width: 367px;
+  height: 48px;
+  background: white;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  margin: 15px 20px 15px 20px;
+}
+
+.text-email {
+  color: #191765;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+}
+
+textarea {
+  width: 367px;
+  height: 148px;
+  margin-right: 10px;
+  margin-left: 10px;
+  margin-top: 15px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+}
+
+.submit-btn {
+  display: block;
+  margin: auto;
+  background: #7d73fc;
+  width: 85px;
+  height: 45px;
+  color: #edecff;
+  font-size: 12px;
+  font-family: "Roboto", sans-serif;
+  margin-top: 30px;
+  padding: 10px;
+  border-color: #7d73fc;
+}
```

Comment:
Great job team ✋

- [ ] [Optional] As required in the project requirements, you may please implement the button interactions (hover and enable) on these buttons. You may as well add the `cursor: pointer` property to add UX to the design.